### PR TITLE
Add Tiles example to Readme

### PR DESCRIPTION
**Pull request by volkansahin45**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,6 @@ Samples
 
 * **[WearStandaloneGoogleSignIn](WearStandaloneGoogleSignIn)** (Java) - Demonstrates using Google sign-in to authenticate user. [Guide](https://developer.android.com/training/wearables/apps/auth-wear)
 
+* **[WearTilesKotlin](WearTilesKotlin)** (Kotlin) - Demonstrates tiles using the new androidX alpha library. [Guide](https://developer.android.com/training/articles/wear-tiles)
```
Comment (alexvanyo) [inline README.md:34]: Super small nitpick, let's use the `AndroidX` capitalization:

```suggestion
* **[WearTilesKotlin](WearTilesKotlin)** (Kotlin) - Demonstrates tiles using the new AndroidX alpha library. [Guide](https://developer.android.com/training/articles/wear-tiles)
```

**Review by alexvanyo — APPROVED** [review]
Ah, totally missed that this wasn't linked in the README. Thank you!

File: README.md (hunk 1 above)
Comment (volkansahin45) [reply README.md]: 👍🏻

**Comment by alexvanyo** [comment]
Thanks @volkansahin45 !

**Pull request merged**